optuna_dashboard/static/components/GraphSlice.tsx

```diff
@@ -171,6 +188,10 @@ const plotSlice = (
       linewidth: 5,
       gridcolor: "#f2f5fa",
       gridwidth: 1,
+      tickfont: {
+        color: "#000000",
+      },
```

Comment:
[nits] Could you tell me what is this attribute for?

Reply:
This makes sure that the color of the x labels is black. It might not be strictly required, but since it was present in the non-numerical case, I added it here for consistency. If you want I can remove it in both places.

Reply:
By the way, there is a lot of duplicate code in the if(isnum) case and the else case. These cases basically only differ in the trace and the code could be simplyfied. I could do that if you want

Reply:
I have another question about the code. On line 177 you have `xaxis: selected` and on line 210 you have `// xaxis: paramName`. So in one case (isnum) it is set and in the other case it is not. What is the effect of this parameter? I did not notice any difference when I commented it out, is it required?

Reply:
> This makes sure that the color of the x labels is black. It might not be strictly required, but since it was present in the non-numerical case, I added it here for consistency. If you want I can remove it in both places.

I see. You are right 👍 

> By the way, there is a lot of duplicate code in the if(isnum) case and the else case. These cases basically only differ in the trace and the code could be simplyfied. I could do that if you want

Sounds good! Welcome.

Reply:
I'll make these changes in a separate PR because I don't know when I have time. Is everything in the current PR ok?

Reply:
> I'll make these changes in a separate PR because I don't know when I have time. Is everything in the current PR ok?

Sure!